rafs/src/metadata/layout/v6.rs

```diff
@@ -182,14 +189,34 @@ impl RafsV6SuperBlock {
             return Err(einval!("invalid extended slots in Rafsv6 superblock"));
         }
 
-        if self.s_blocks == 0 {
-            return Err(einval!("invalid blocks in Rafsv6 superblock"));
-        }
-
         if self.s_u != 0 {
             return Err(einval!("invalid union field in Rafsv6 superblock"));
         }
 
+        if self.s_xattr_blkaddr != 0 {
+            return Err(einval!(
+                "invalid extended attribute address in Rafsv6 superblock"
```

Comment:
`unsupported shared extended attribute namespace in Rafsv6 superblock`

it indicates that it could be supported in Nydus in the future